scormxblock/static/js/src/scormxblock.js

```diff
@@ -125,6 +128,42 @@ function ScormXBlock_${block_id}(runtime, element) {
       return false;
     }
 
+    function disableNextModuleArrow() {
+      if (isNewUI()) {
+        disableNextModuleArrowForNewUI();
+      }
+      else {
+        disableNextModuleArrowForOldUI();
+      }
+    }
+
+    function isNewUI() {
+      return $( "body.new-theme" ).length;
```

Comment:
I believe this will give undefined if OldUi is active. Can we return boolean here instead?
$(".blob-code.blob-code-empty.empty-cell").length?true:false

Reply:
`$("body").hasClass("new-theme")` would do the job.

Reply:
This will return 0 in case of oldUI.